acceptance/tests/pxp_module_bolt_command/run_command.rb

```diff
@@ -17,7 +19,7 @@
   end
 
   step 'Run a command (`echo`) on agent hosts' do
-    agents.each do |agent|
+    suts.each do |agent|
```

Comment:
I think we should still run this against all the agent platforms?

Reply:
Yeah, that's still happening.

This `suts` bit was added because `agents` actually includes the master's agent and not just the endpoint agents. Technically that means each of these tests would run twice per cell: once for the actual agent and again for the master agent.

Reply:
ohhh, derp its `reject` my bad. I thought you were filtering on anything that matched `master`.